### PR TITLE
JAVA-27740 Fix failing module jhipster-6 - upgrade to JDK 17

### PR DESCRIPTION
**Pull request by timis1**

### CODE REVIEW

File: jhipster-6/bookstore-monolith/pom.xml [hunk 1]
```diff
@@ -1125,6 +1125,8 @@
     <properties>
         <!-- Build properties -->
         <maven.version>3.0.0</maven.version>
```
Comment (anastasiosioannidis) [inline jhipster-6/bookstore-monolith/pom.xml:1127]: Is explicitly setting these needed?
They should be inherited from the parent pom.
Try to remove these and check that module is build on JDK 17

Comment (timis1) [reply jhipster-6/bookstore-monolith/pom.xml:1127]: I didn't touch the old settings. I will remove and try to rebuild

**Comment by anastasiosioannidis** [comment]
retest this please

**Comment by timis1** [comment]
> retest this please

@anastasiosioannidis What exactly do you want to test?

**Comment by timis1** [comment]
Test

**Comment by asif-anwar** [comment]
retest this please

**Comment by timis1** [comment]
retest this please

**Comment by anastasiosioannidis** [comment]
retest this please

**Pull request merged**